internal/handlers/image_handler.go

```
@@ -0,0 +1,283 @@
+}
+
+func (h *ImageHandler) DeleteImage(c *gin.Context) {
+	imageID := c.Param("id")
+
+	var req struct {
+		Force   bool `json:"force"`
+		NoPrune bool `json:"noPrune"`
+	}
+
+	// Bind query parameters or JSON body
+	c.ShouldBindJSON(&req)
```

golangci-lint (1.64.8)

Error return value of `c.ShouldBindJSON` is not checked

(errcheck)